code/modules/research/techweb/all_nodes.dm

```diff
@@ -180,7 +180,7 @@
 	display_name = "Anomaly Research"
 	description = "Unlock the potential of the mysterious anomalies that appear on station."
 	prereq_ids = list("adv_engi", "practical_bluespace")
-	design_ids = list("reactive_armour", "anomaly_neutralizer")
+	design_ids = list("reactive_armour", "anomaly_neutralizer", "xenoa_labeler", "xenoa_list_console", "xenoa_list_pad", "xenoa_gloves")
```

Comment:
I find it strange that roundstart gear requires research in order to obtain. It can be really annoying if you have multiple people and require late game research to get a new labeller since they can only be used by one person at a time. The gloves make sense, since they give you an advantage and aren't required (even though they are just available in the spawn room) but restricting gear that is required for a roundstart science module to late game research is annoying.

Reply:
Moved labeler to base